daml-foundations/daml-tools/da-hs-daml-cli/tests/VisualTest.hs

```diff
@@ -0,0 +1,39 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+{-# LANGUAGE OverloadedStrings #-}
+module VisualTest
+   ( main
+   ) where
+
+import Test.Tasty
+import Test.Tasty.HUnit
+import System.Directory
+import System.IO.Extra
+import DA.Cli.Visual
+
+
+main :: IO ()
+main = defaultMain tests
+
+tests :: TestTree
+tests = testGroup "Tests" [unitTests]
+
+unitTests :: TestTree
+unitTests = testGroup "Unit tests"
+  [
+    testCase "file exists" (doesFileExist "daml-foundations/daml-tools/da-hs-daml-cli/visual-test-daml.dar" @? "missing dar file to process") ,
+    testCase "file exists" (doesFileExist "daml-foundations/daml-tools/da-hs-daml-cli/tests/res/out.dot" @? "missing the expected dot file to check") ,
+    testCase "dot file check" $ do
+        withTempFile $ \path -> do
+            let darPath = "daml-foundations/daml-tools/da-hs-daml-cli/visual-test-daml.dar"
+            _ <- execVisual darPath (Just path)
+            shouldThrow path "daml-foundations/daml-tools/da-hs-daml-cli/tests/res/out.dot"
```

Comment:
This seems a really odd test and a really odd way to assert a file doesn't exist

Reply:
removed the redundant tests